### PR TITLE
Update structure + format of DDG goodie

### PR DESCRIPTION
**Pull request by bsstoner**
To address #853

I restructured things according to @abeyang's mockup and got the first one ("about") working.

We need to finish going through all the responses and rewrite the text so it makes sense in the new layout. I could use some help if anyone's up for it: https://github.com/duckduckgo/zeroclickinfo-goodies/blob/892f9ea0c0cacfbff6faccbb9a1089a24ce8b178/share/goodie/duck_duck_go/responses.yml

After that, just need to update the tests!

@jagtalon @moollaza @chrismorast

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/DuckDuckGo.pm [hunk 1]
```diff
@@ -58,11 +45,22 @@ handle remainder => sub {
...
+        id => 'duck_duck_go',
+        data => {
+            title => $response->{title},
+            subtitle_text => $response->{subtitle},
+            subtitle_url => $response->{url}
+        },
+        templates => {
+            group => 'text',
+            options => {
+                subtitle_content => 'DDH.duck_duck_go.subtitle_content'
```
Comment (moollaza) [inline lib/DDG/Goodie/DuckDuckGo.pm]: We should be able to create the url if we pass in an object thanks to our `subtitle` helper:

```perl
subtitle => { url => $response->{url}, text => $response->{subtitle}, className => "tx--14" }
```

I need to confirm this still...

Comment (moollaza) [inline lib/DDG/Goodie/DuckDuckGo.pm]: Yup, it works :smile:

Comment (moollaza) [inline lib/DDG/Goodie/DuckDuckGo.pm]: Ah, but we can't add in the favicon this way...

**Comment by moollaza** [comment]
I've done a first pass here: https://gist.github.com/moollaza/eefed2b30803b1d14529 -- open to feedback/suggestions

//cc @zekiel

**Comment by bsstoner** [comment]
I merged @moollaza's gist and made some revisions. Some of these feel outdated and there's probably a bunch that are missing, i.e. "duckduckgo slack", "duckduckgo meetups", etc.. But even if it's not perfect, I think it's a big improvement over what's in prod currently.

@zekiel or others, feel free to jump in and add revisions: https://github.com/duckduckgo/zeroclickinfo-goodies/blob/brian/duckduckgo-goodie/share/goodie/duck_duck_go/responses.yml

File: lib/DDG/Goodie/DuckDuckGo.pm (hunk 1 above)
Comment (bsstoner) [inline lib/DDG/Goodie/DuckDuckGo.pm]: yea, so annoying... trying to do this PR made me realize how difficult the templates are to work with.

Comment (moollaza) [inline lib/DDG/Goodie/DuckDuckGo.pm]: I'm both happy and sad to hear this :+1:

**Comment by bsstoner** [comment]
:+1: Thanks @zekiel, just verified links with ids like that work.

Anyone else want to make revisions? IMO the fact that the links are clearer is big enough improvement to just merge and we can iterate on any other content changes.

@moollaza / @jagtalon want to do a final review and merge?

**Comment by jagtalon** [comment]
@bsstoner checking it out!

**Comment by jagtalon** [comment]
@bsstoner Ugh, sometimes an oxford comma is used, and sometimes it isn't. What are we, barbarians?!

I kid. It looks good! :+1: 

@moollaza What do you think about adding the edge browser?

**Pull request merged**